graveler/sstable/cache.go

```diff
@@ -3,25 +3,35 @@ package sstable
 import (
 	"fmt"
 
-	"github.com/cockroachdb/pebble/sstable"
 	lru "github.com/treeverse/lakefs/cache"
-	"github.com/treeverse/lakefs/graveler/committed"
 	"github.com/treeverse/lakefs/pyramid"
+
+	"github.com/cockroachdb/pebble/sstable"
 )
 
 //go:generate mockgen -source=cache.go -destination=mock/cache.go -package=mock
 
 type Derefer lru.Derefer
 
-type opener func(namespace string, id committed.ID) (Item, error)
+type opener = func(namespace string, filename string) (Item, error)
+
+type existser = func(namespace string, filename string) (bool, error)
 
-type cache interface {
-	GetOrOpen(namespace string, id committed.ID) (*sstable.Reader, Derefer, error)
+type ID string
```

Comment:
Why do we need another `ID`? synced f2f and `sstable` should be allowed to import `committed`.